src/decanter/core/jobs/experiment.py

```diff
@@ -48,7 +49,9 @@ class Experiment(Job):
     """
     def __init__(self, train_input, select_model_by='mse', name=None):
         super().__init__(
-            jobs=[train_input.data],
+            jobs=[train_input.data, train_input.data.jobs[0]] 
```

Comment:
After adding the correct dependency between DataSetup and DataUpload jobs,
Experiment job will have the denpendency of corresponded data.
So threre should be no changes hrere.